app-simple-wallet/app/src/main/java/com/goldenraven/devkitwallet/ui/wallet/ReceiveScreen.kt

```diff
@@ -54,7 +54,7 @@ internal class AddressViewModel : ViewModel() {
         get() = _addressIndex
 
     fun updateAddress() {
-        _address.value = Wallet.getLastUnusedAddress().address
+        _address.value = Wallet.getLastUnusedAddress().address.toString()
```

Comment:
The toString method will produce this when redered. I think what you meant to use is the [asString](https://bitcoindevkit.org/android/bdk-android/org.bitcoindevkit/-address/index.html?query=class%20Address(address:%20String)) method (I know they're annoyingly close in name 😆)
<img width="360" alt="tostring" src="https://github.com/thunderbiscuit/devkit-wallet/assets/39974688/bacea8f5-738a-4105-9059-9ca91cdd9784">